content/applications/websites/ecommerce/products/additional_features.rst

```diff
@@ -0,0 +1,157 @@
+===================
+Additional features
+===================
+
+Odoo eCommerce offers some additional :ref:`buttons
+<ecommerce/products/additional_features/buttons>` and :ref:`features
+<ecommerce/products/additional_features/product-highlight>` that can be enabled and used throughout
```

Comment:
```suggestion
Odoo eCommerce offers additional :ref:`buttons
<ecommerce/products/additional_features/buttons>` and :ref:`product highlight
<ecommerce/products/additional_features/product-highlight>` feature that can be enabled and used throughout
```
Since there is only one feature (so far) we could name it directly.